dojo_plugin/utils/dojo.py

```diff
@@ -63,6 +65,8 @@
         **VISIBILITY,
 
         Optional("image"): IMAGE_REGEX,
+        Optional("allow_privileged"): bool,
+        Optional("unimportable"): bool,
```

Comment:
Let's change `unimportable` to `importable` (with a default of `True`, of course). I know that using this field explicitly means you're going to be disabling importability, but spending any time thinking about a double negative is just always annoying. I think `importable: False` is much more readable than `unimportable: True`.

One other naming thought would be `public_importable` if we think there's any realistic chance in the future we need dojo admins to be able to import from one of their other dojos which they are an admin in, but no one else should be able to. For example, if we actually end up using this in a belted dojo, and then want to use it in a course dojo. Some bypass (probably this outlined common admin idea) would be necessary.